index.html

```diff
@@ -266,7 +277,8 @@ <h2 class="contact-me-text">Contact me</h2>
                             cols="30"
                             required>
                         </textarea> 
-                        <button type="submit">Get in Touch</button>    
+                        <span class="error" aria-live="polite"></span> 
+                        <button type="submit" id="submit">Get in Touch</button>   
```

Comment:
Great job team 🤝

- [ ] As specified in the project requirements, you may kindly align your button accordingly. Please check the design provided in the Figma template.
- Your design
![align button](https://user-images.githubusercontent.com/63917725/180310717-62db0ef9-336c-4365-b7b0-4c4adcd2f2ca.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/180311190-23c219de-90b1-40e0-ad8b-08e05e44eb2c.PNG)